Fantom/fantom-memecoins/src/mappings/mappingHandlers.ts

```diff
@@ -0,0 +1,64 @@
+import { Approval, Transfer, Address } from "../types";
+import {
+  ApproveTransaction,
+  TransferLog,
+} from "../types/abi-interfaces/Erc20Abi";
+import assert from "assert";
+import { Erc20Abi__factory } from "../types/contracts";
+import { contractAddress } from "../const";
+
+const erc20 = Erc20Abi__factory.connect(contractAddress, api);
+
+async function getOrCreateAddress(accountAddress: string): Promise<Address> {
+  let address = await Address.get(accountAddress);
+
+  if (address == undefined) {
+    address = Address.create({
+      id: accountAddress,
+      balance: (await erc20.balanceOf(accountAddress)).toBigInt(),
```

Comment:
Should you also update the balance of an address if it's known (e.g. refresh this on each new transaction?)

Reply:
Ahh I see you do this below